internal/data_generator/internal/trace_structs.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package internal
+
+var traceFile = &File{
+	Name: "trace",
+	imports: []string{
+		`otlptrace "github.com/open-telemetry/opentelemetry-proto/gen/go/trace/v1"`,
+	},
+	structs: []baseStruct{
+		spanEvent,
+		//SpanLink,
+		spanStatus,
+	},
+}
+
+var spanEvent = &messageStruct{
+	structName: "SpanEvent",
+	description: "// SpanEvent is a time-stamped annotation of the span, consisting of user-supplied\n" +
```

Comment:
nit: it seems that the generator already adds "// SpanEvent" see stutter on generated code. Same for the next struct.

Reply:
I fixed that in #670. Will get regenerated this file.